docs/build_and_load.md

```diff
@@ -0,0 +1,766 @@
+Next, we must define ordered sequences of genic and inter-genic 
+ranges across the reference genome. These ordered ranges, which we 
+will call "reference ranges", are used to specify haplotype sequence 
+coordinate information for the haplotype and genomic variant call 
+format data. These genomic positions are structured using the 
+[BED](https://en.wikipedia.org/wiki/BED_(file_format)) format.
+
+Generally, this data is not readily available as BED files. PHGv2
+can generate this file directly from 
+[GFF](https://en.wikipedia.org/wiki/General_feature_format)-formatted 
+data using the `create-ranges` command:
+
+```shell
+./phg create-ranges \
+    --gff data/anchors.gff \
+    --reference-file data/Ref.fa \
+    --boundary gene \
+    --pad 500 \
+    -o output/ref_ranges.bed
+```
+
+This command uses several parameters:
+
+* `--gff` - Our reference GFF file of interest. Since 
+  reference ranges are usually established by genic/intergenic 
+  regions, we can leverage coordinate and feature information from
+  GFF files.
+* `--reference-file` - Our reference genome in 
+  [FASTA](https://en.wikipedia.org/wiki/FASTA_format) format. This
+  is needed for determining the intergenic regions near the ends of 
+  chromosomes.
+* `--boundary` - How do you want to define the boundaries of the 
+  reference ranges. Currently, these can be defined as either
+  `gene` or `cds` regions:
+
+  <img src="img/build_and_load/create_ranges_01.svg" width="500" alt=""/>
+  
+  + In the above figure, if `--boundary` is set to `gene`, the start
+    and end positions are at the UTR regions for each gene feature from
+    the GFF file, while `cds` will begin and end at the open reading 
+    frame. By default, the `cds` option will try to identify the
+    transcript with the longest open reading frame if there are
+    multiple transcript "children" for a gene "parent" in the GFF
+    file.
+* `--pad` - The number of base pairs you would like to flank regions:
+
+  <img src="img/build_and_load/create_ranges_02.svg" width="500" alt=""/>
+
+  + For example, if we were to set the `--pad` parameter to `500`,
+    we would extend the region 500 base pairs upstream and downstream
+    of the defined boundary (in this case, `gene`).
+
+> [!NOTE]
+> There is a possibility that overlaps of regions will occur. If
+> this does happen, `create-ranges` will identify any overlapping
+> regions and merge regions together:
+> ```
+> [---Overlap 1----]
+>            [-------Overlap 2-------]
+> ====================================
+> [---------Merged overlap-----------]
+>  ```
+  
+* `-o` - Name for the output BED file.
+
+In the above example, I am using test data from the 
+[PHGv2 GitHub repository](https://github.com/maize-genetics/phg_v2/tree/main/data/test/smallseq).
+After the command is finished, we have produced a BED file (which
+in my case, is located in a subdirectory labelled `output`). This BED 
+file contains 4 columns of information:
```

Comment:
Looking at the code, there are actually 6 columns returned.  contig, start, end, name, score and strand.  The "name" is the gene or cds name.  If regions overlapped and were merged, the name is a comma separated string of all the genes/cds included in this bed region.  THe score appears to always be 0 - not sure what that is supposed to represent (Zack would know) nor am I sure why strand is included as I thought bed files were always on the forward strand.